pkg/backend/openshiftcluster/openshiftcluster.go

```diff
@@ -51,44 +51,44 @@ type manager struct {
 }
 
 // NewManager returns a new openshiftcluster Manager
-func NewManager(log *logrus.Entry, _env env.Interface, db database.OpenShiftClusters, cipher encryption.Cipher, billing billing.Manager, doc *api.OpenShiftClusterDocument, subscriptionDoc *api.SubscriptionDocument) (Manager, error) {
-	localFPAuthorizer, err := _env.FPAuthorizer(_env.TenantID(), _env.Environment().ResourceManagerEndpoint)
+func NewManager(log *logrus.Entry, env env.Interface, db database.OpenShiftClusters, cipher encryption.Cipher, billing billing.Manager, doc *api.OpenShiftClusterDocument, subscriptionDoc *api.SubscriptionDocument) (Manager, error) {
```

Comment:
We still have a number of other places where we use `_env` as a pattern. I would like to keep it as it is with `_env` considering the package is names same `env`. Which is causing confusion and overload of the same word/term.

Reply:
Discussed this - going to stay with current convention of only using `_env` as a name if necessary.